spec/unit/sandbox/file_accessor_spec.rb

```diff
@@ -237,7 +237,7 @@ module Pod
           file_patterns = ['Classes/*.{h,m,d}', 'Vendor', 'framework/Source/*.h']
           options = {
             :exclude_patterns => ['Classes/**/osx/**/*', 'Resources/**/osx/**/*'],
-            :dir_pattern => '*{.m,.mm,.i,.c,.cc,.cxx,.cpp,.c++,.swift,.def,.h,.hh,.hpp,.ipp,.tpp,.hxx}',
+            :dir_pattern => '*{.m,.mm,.i,.c,.cc,.cxx,.cpp,.c++,.swift,.h,.hh,.hpp,.ipp,.tpp,.hxx}',
```

Comment:
Actually because `file_accessor_spec` is already merged like this, it wont show up in the diff.